VL.Data/Profiles/BookProfile.cs

```diff
@@ -0,0 +1,31 @@
+﻿using AutoMapper;
+using VL.Shared.Model;
+
+namespace VL.Shared.Profiles
+{
+    public class BookProfile : Profile
+    {
+        public BookProfile()
+        {
+            CreateMap<UpdateBookDto, Book>()
+                .ForMember(
+                    dest => dest.Id,
+                    from
+                        => from.MapFrom(x => x.Id)
+                        )
+                .ForMember(
+                    dest => dest.Author,
+                    from
+                        => from.MapFrom(x => x.Author)
+                        )
+                .ForMember(dest => dest.Title,
+                    from
+                        => from.MapFrom(x => x.Title)
+                        )
+                .ForMember(dest => dest.Description,
+                    from
+                        => from.MapFrom(x => x.Description)
+                        );
```

Comment:
Is any of this required?

Reply:
It's the way I did it?

Reply:
Specifying .ForMember on the mapping profile where the source name equals the destination name, is that required

Reply:
Not if all of the names are the same like they are in mine

Reply:
That's right - so we can remove the for member and use automapper by default convention